frontend/src/store/shoot/index.js

```diff
@@ -441,6 +443,7 @@ export const useShootStore = defineStore('shoot', () => {
 
     state.newShootResource = value
     state.initialNewShootResource = cloneDeep(value)
+    shootStagingStore.workerless = false
```

Comment:
maybe introduce a reset function on shootStagingStore?

Reply:
As discussed, we could do this. Right now the shootStagingStore is only used by components that set the state when they are initialized. `workerless` is the only value that needs to be reset as it is not represented by a shoot spec attribute. It is used to back the state of the checkbox only.
However, I'm not against introducing a reset method.